packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -107,9 +108,22 @@ const styles = (theme: ThemeType) => ({
     marginLeft: 'auto',
     marginRight: 10
   },
-  dialogueMatchPreferencesButton: {
+  dialogueMatchPreferencesButtonContainer: {
     marginLeft: 8,
-    marginRight: 0
+    marginRight: 0,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'flex-end',
+    position: 'relative',
+  },
+  enterTopicsAnnotation: {
+    fontStyle: 'italic',
+    color: theme.palette.text.dim3,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    "fontSize": "0.9rem",
+    "lineHeight": "1rem",
```

Comment:
these don't need to be in quotes

Reply:
Hm, I am getting errors trying to remove the quotes. Also many other places in the codebase wher ethey are in quotes. 

Might grab you later to teacm me what's up, will resolve for now

Reply:
I meant the property names, not the values

Reply:
fixed